src/main/groovy/com/smartbear/swagger/Swagger2TestCaseGenerator.groovy

```diff
@@ -0,0 +1,60 @@
+package com.smartbear.swagger
+
+import com.eviware.soapui.impl.rest.RestRequestInterface.HttpMethod
+import com.eviware.soapui.impl.rest.RestResource
+import com.eviware.soapui.impl.wsdl.WsdlProject
+import com.eviware.soapui.impl.wsdl.WsdlTestSuite
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.impl.wsdl.teststeps.registry.RestRequestStepFactory
+import com.eviware.soapui.model.testsuite.TestProperty
+import io.swagger.models.Operation
+import io.swagger.models.Path
+import io.swagger.models.parameters.AbstractSerializableParameter
+
+class Swagger2TestCaseGenerator {
+    static void generateTestCases(WsdlProject project, RestResource resource, Path path) {
+        WsdlTestSuite testSuite = createTestSuiteIfNotPresent(project)
+        WsdlTestCase testCase = testSuite.addNewTestCase("$resource.name-TestCase")
+        resource.restMethodList.each {
+            method ->
+                method.requestList.each {
+                    request ->
+                        RestTestRequestStep testStep = (RestTestRequestStep) testCase.addTestStep(RestRequestStepFactory.createConfig(request, request.name))
+                        testStep.testRequest.params.each {
```

Comment:
we need to handle body parameters in the Swagger-definition - i.e. POST/PUT bodies that have an associated model defined - this is handled in the Swagger2Importer when it creates default requests so we could probably share code with it